packages/stats/gbstats/bayesian/tests.py

```diff
@@ -79,12 +104,15 @@ def __init__(
         self,
         stat_a: TestStatistic,
         stat_b: TestStatistic,
-        inverse: bool = False,
-        ccr: float = 0.05,
+        config: BayesianConfig = BayesianConfig(),
     ):
         super().__init__(stat_a, stat_b)
-        self.ccr = ccr
-        self.inverse = inverse
+        self.ccr = config.ccr
+        self.inverse = config.inverse
+        self.relative = config.difference_type == "relative"
```

Comment:
We should remove these 4 lines (relative, scaled, traffic_proportion_b, phase_length_days) from `Gaussian` and `Binomial` BayesianABTest classes. They are redundant now.

Reply:
done